docker-compose.malware-scanner.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+
+  clamav:
+    image: clamav/clamav:stable_base
+    container_name: clamav
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/malware-scanner/clamav/defs:/var/lib/clamav
+      - ./docker/malware-scanner/clamd.conf:/etc/clamav/clamd.conf:ro
+    expose:
+      - 3310 # NEVER expose this outside of the local network!
+    deploy:
+      resources:
+        limits:
+          cpus: '0.50'
+    networks:
+      shared:
+        ipv4_address: 10.10.10.100
+
+  malware-scanner:
+    build:
+      context: ./docker/malware-scanner
+      dockerfile: Dockerfile
+      args:
+        branch: ivo/impl
```

Comment:
this is a temporary branch right? should be main at some point once it's all finished?

Reply:
Yes, I'll change it to `main` once it's merged.

Reply:
Since we're not in a great hurry to merge this, I would leave this PR unmerged until the malware-scanner is merged and we can point this to the `main` branch. It shouldn't take too long.